testament/important_packages.nim

```diff
@@ -11,7 +11,7 @@ var packages2*: seq[tuple[name, cmd: string; hasDeps: bool; url: string, useHead
 # packages A-M
 # pkg1 "alea", true
 pkg1 "argparse"
-pkg1 "arraymancer", true, "nim c tests/tests_cpu.nim"
+#pkg1 "arraymancer", true, "nim c tests/tests_cpu.nim" # See https://github.com/nim-lang/Nim/pull/14862#issue-442239265
```

Comment:
Your comment links to the main PR comment, but you don't explain why this needed to be commented out?